### DISCUSSION TITLE
something is wrong with max_concurrent

### DISCUSSION THREAD

**Discussion by shafran666** [swar/Swar-Chia-Plot-Manager#140] [Q&A]
What am i doing wrong? I’ve got corei7 10700K (8/16 cores/threads), 64GB RAM and temp.drive: raid-2x nvme m2 2TB.
I wanted to do simultaneously10 plots, max 3 per phase one, this is the config:

```
- name: main
max_plots: 54
farmer_public_key:
pool_public_key:
temporary_directory: D:\
temporary2_directory:
destination_directory: F:\
size: 32
bitfield: true
threads: 4
buckets: 128
memory_buffer: 6000
max_concurrent: 10
max_concurrent_with_start_early: 12
stagger_minutes: 0
max_for_phase_1: 3
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false
```
and i constantly get 8:
![image](https://user-images.githubusercontent.com/83636793/117416968-2e8dfd80-af1a-11eb-9621-6e5890c90a69.png)

sometimes even without 3 plots on phase 1:
![image](https://user-images.githubusercontent.com/83636793/117417086-4b2a3580-af1a-11eb-8316-8385a2b3d918.png)

**Comment by Drummas** [comment]
Late reply as you probably solved it already, but might be useful for others.

Your being core capped.

4 threads per plot, 3 plots on phase 1 = 12 threads which leaves 4 for plots on other phases, meaning you can only do 8 max concurrent plots, not 10.

You should be able to do 10 if you set it to 3 threads per plot.

**Reply by Arkex-3** [reply]
just a question here
if he's using 12 threads bc of plotting 3 in p1 with  4 each
he should only be able to to 4 plots additional by using 1 Thread for each at the other phases
so its a total of 7 Plots parallel?

**Reply by Drummas** [reply]
Correct, he should be able to run 7, not 8 as I previously stated, but most of the times you can actually squeeze an extra plot, no idea why. If I had to guess, I'd say the CPU just distributes the load of that extra plot amongst any threads that are not at 100% utilization at the time

**Reply by Cminer3207** [reply]
Check your Global Settings above the job-orders...
